### DISCUSSION TITLE
"Number" card attributes should include prefix/suffix when enabled

### DISCUSSION THREAD

**Discussion by bryantwells** [craftcms/cms#16336] [Ideas] [answered]
For example, i have a number field that has a suffix of "inches". When displayed in a card preview, this suffix should be visible as well.

![Screen Shot 2024-12-15 at 00 01 00](https://github.com/user-attachments/assets/09d96d3e-661f-4e15-b374-cec2a2ffc9d8)

**Comment by brandonkelly** [comment, marked as answer]
Thanks for the suggestion! Craft 5.7.0 is out with that change (#16986).
